ci-operator/step-registry/baremetalds/two-node/fencing/post-install/node-degredation/baremetalds-two-node-fencing-post-install-node-degredation-commands.sh

```
@@ -84,92 +175,58 @@ else
-sudo pcs status || true
-sudo pcs resource status || true
-PCS_EOF
-fi
 EOF
   ssh_rc=${PIPESTATUS[0]}
   set -e
 
   if [[ ${ssh_rc} -ne 0 ]]; then
-    log "ERROR: Failed to degrade ostest_master_1 via hypervisor (rc=${ssh_rc})"
+    log "ERROR: Failed to degrade ${VICTIM_NODE} (${VICTIM_VM}) via hypervisor (rc=${ssh_rc})"
     exit ${ssh_rc}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)